storage-proofs/src/vde.rs

```diff
@@ -1,4 +1,5 @@
 use blake2s_simd::Params as Blake2s;
```

Comment:
Trying to comment on the file name: Can we please finally rename this from `vde`, even if we don't kill all remaining sloths (Slothfather forgive me.)

Reply:
haha, more changes :P